### DISCUSSION TITLE
Does importing modpacks via `FTB App Import` copy the files over or re-use the existing directory?

### DISCUSSION THREAD

**Discussion by mainrs** [PrismLauncher/PrismLauncher#2746] [Q&A] [answered]
The former would prevent PL to pick up updates triggered through the FTB App.

**Comment by maskersss** [comment]
it copies the files over, if you want ig you can try symlinking prism's .minecraft folder to ftb app's .minecraft folder?

**Reply by mainrs** [reply, marked as answer]
Thanks for letting me know!
